src/Button/spec.js

```diff
@@ -0,0 +1,121 @@
+export default [{
+   name: 'borderRadius',
+   type: 'String',
+   default: '4px',
+   description: 'Set the border-radius of the button. By default, all buttons inherit their borderRadius from the <code>elementBorderRadius</code> prop set on the parent <code>OIO</code> container.',
+   responsive: true
+}, {
+   name: 'className',
+   type: 'String',
+   default: '-',
+   description: 'Pass a <code>className</code> to customize className of the component',
+   responsive: false
+}, {
+   name: 'color',
+   type: 'String',
+   default: '-',
+   description: 'The primary button color (usually its background color). Pass any css compatible color value. By default, all buttons inherit their color from the <code>highlightColor</code> prop set on the parent <code>OIO</code> container.',
+   responsive: true
+}, {
+   name: 'fontFamily',
+   type: 'String',
+   default: '-',
+   description: 'Apply a specific font-family to the text within this component. By default, all buttons inherit the <code>fontFamily</code> set on the parent <code>OIO</code> container',
+   responsive: true
+}, {
+   name: 'id',
+   type: 'String',
+   default: '-',
+   description: '',
+   responsive: false
+}, {
+   name: 'mode',
+   type: 'String (Enum)',
+   default: '<code>normal</code>',
+   description: 'Availalble options include: <code>disabled</code>, <code>normal</code>, <code>pulsing</code>, <code>loading</code>.<br /><br /><b>Note</b>: When mode is set to <code>loading</code>, the <code>Button</code> will retain the same size were its <code>mode</code> set to <code>normal</code>',
```

Comment:
Hmm just a small thing, which is that the size only remains the same if `children` and `name` do not change when the mode changes to/from `loading`.